packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -114,6 +115,14 @@ export namespace CommonCommands {
         id: 'core.saveAll',
         label: 'Save All'
     };
+    export const ENABLE_AUTO_SAVE: Command = {
+        id: 'textEditor.commands.autosave.enable',
+        label: 'Enable Auto Save'
+    };
+    export const DISABLE_AUTO_SAVE: Command = {
+        id: 'textEditor.commands.autosave.disable',
+        label: 'Disable Auto Save'
+    };
```

Comment:
@vinokurig is there a reason not to have just one command - `Toggle Auto Save`?
I think it'd be more convenient.

Reply:
Sorry, haven't seen your previous comment.